caught exception: "catch me"
done.
